### DISCUSSION TITLE
Error: A black box formula is still use in the entity "Partition coefficient" in Mobi

### DISCUSSION THREAD

**Discussion by Renchugo** [Open-Systems-Pharmacology/Forum#1616] [Questions & Problems] [answered]
Hi all,

I am new to the PK-sim/Mobi and just want to explore how to extend sturcture in Mobi.

I want to replicate the bird model in: https://github.com/Open-Systems-Pharmacology/Birds-PBKModel

When I run the Chicken_RR in Mobi, I got some errors

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/105068944/892fb9d5-2505-4251-a87a-8d2af7999210)

Is there anything I am missing? Highly appreciate to any suggestions.

Thank you!

**Comment by Renchugo** [comment]
Sorry, the updated link for the bird model is:
https://github.com/Open-Systems-Pharmacology/Birds-PBK-Model/tree/main/Code/Models/Simulations

**Comment by PavelBal** [comment, marked as answer]
Thank you for pointing this out. We have updated the description of the repository, please read section https://github.com/Open-Systems-Pharmacology/Birds-PBK-Model/blob/main/README.md#running-the-simulations

**Reply by Renchugo** [reply]
Thank you Pavel! It works. 👍

**Reply by Xiaoyang-svg** [reply]
> Thank you Pavel! It works. 👍
Hello Renchugo, I also wanted to replicate the bird model and copied the AllCalculationMethods.pkml file into the location of MoBi 9.1, but I got some errors below:
![image](https://github.com/user-attachments/assets/f4c7d9e2-21ee-421c-bbe8-3f652cae41e5)
Have you ever seen a problem like this before? Thank you very much and look forward to your reply.
